Packs/CommonScripts/Scripts/SearchIncidentsV2/SearchIncidentsV2_test.py

```diff
@@ -211,3 +230,50 @@ def test_summarize_incidents():
     assert summarize_incidents({'add_fields_to_summarize_context': 'test'}, [{'id': 'test', 'CustomFields': {}}]) == [
         {'closed': 'n/a', 'created': 'n/a', 'id': 'test', 'incidentLink': 'n/a', 'name': 'n/a', 'owner': 'n/a',
          'severity': 'n/a', 'status': 'n/a', 'test': 'n/a', 'type': 'n/a'}]
+
+
+@pytest.mark.parametrize('amount_of_mocked_incidents, args', [
+    (306, {}),
+    (306, {"limit": "200"}),
+    (105, {"limit": "200"}),
+    (1000, {"limit": "100"}),
```

Comment:
lets also add in a not round number like 55, to make sure that we get back no more than 55